src/django_components/component.py

```diff
@@ -588,7 +593,8 @@ def render(
               or to the end of the `<body>` tag. CSS dependencies are inserted into
               `{% component_css_dependencies %}`, or the end of the `<head>` tag.
         - `render_dependencies` - Set this to `False` if you want to insert the resulting HTML into another component.
-
+        - `request` - The request object. This is only required when needing to use RequestContext,
```

Comment:
We should add that `request` is ignored when `context` is given and it is already an instance of `Context` (so we don't have to initialize `Context`.

Reply:
And we should also update the docs in src/docs/concepts/fundamentals/components_in_python.md. Tho note that that file will be moved in https://github.com/EmilStenstrom/django-components/pull/816